db/migrate/20120930040540_create_users.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :name
       t.string :token
 
-      t.timestamps
+      t.timestamps null: false
```

Comment:
@jasnow Rails is pretty good at handling `timestamps` for us, any specific reason to change this old migrations? (isn't even reflected on `schema.rb`)